README.md

```diff
@@ -12,8 +12,8 @@ notifications** to `FCM` (Firebase Cloud Messaging) and/or
 
 #### Running from DockerHub
 
-We provide already built MongoosePush images. If you just want to use it, then all you need is `docker`, `FCM` app token and/or `APNS` app certificates.
-In case of certificates and token you need to setup the following directory structure:
+We provide already built MongoosePush images. If you just want to use it, then all you need is `docker`, `FCM` app token and/or `APNS` app certificates or token. `APNS` can be configured with either of these - you only need one.
+For the full configuration you need to setup the following directory structure:
```

Comment:
For the full configuration you need to set the following directory structure up: